ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/TaskList/Decision/DecisionDateIntegrationTests.cs

```diff
@@ -38,16 +44,17 @@ public async Task Should_persist_approval_date()
 
 			await ProgressToDecisionDateStep(project);
 
-			Document.QuerySelector<IHtmlInputElement>("#decision-date-day").Value = expectedDay;
-			Document.QuerySelector<IHtmlInputElement>("#decision-date-month").Value = expectedMonth;
-			Document.QuerySelector<IHtmlInputElement>("#decision-date-year").Value = expectedYear;
+			DayPart.Value = expectedDay;
+			MonthPart.Value = expectedMonth;
+			YearPart.Value = expectedYear;
+
 			await Document.QuerySelector<IHtmlButtonElement>("#submit-btn").SubmitAsync();
 
 			await OpenUrlAsync($"/task-list/{project.Id}/decision/decision-date");
 
-			var day = Document.QuerySelector<IHtmlInputElement>("#decision-date-day").Value;
-			var month = Document.QuerySelector<IHtmlInputElement>("#decision-date-month").Value;
-			var year = Document.QuerySelector<IHtmlInputElement>("#decision-date-year").Value;
+			var day = DayPart.Value;
```

Comment:
'#decision-date-day' type(s) is in use within  cypress commands, could we have this consistently across all journeys?

Reply:
Yep; nothing has changed in that respect :)